croissant-rdf/tests/test_generate_rdf.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from rdflib import Graph
 
-from croissant_rdf.providers import HuggingfaceHarvester
+from croissant_rdf import HuggingfaceHarvester
 
 OUTPUT_FILEPATH = "./tests/test_output.ttl"
```

Comment:
nit: We should avoid this pattern as pointed out in the initial review. This raises issues when distributing the tests or running them in parallel. Instead we should create [temporary files](https://docs.python.org/3/library/tempfile.html) within the test.

Reply:
The same comment would apply to all tests in this PR.